config/idlenesswatchersettings.cpp

```diff
@@ -72,6 +72,8 @@ void IdlenessWatcherSettings::mConnectSignals()
     connect(mUi->backlightSlider, SIGNAL(valueChanged(int)), SLOT(saveSettings()));
     connect(mUi->idleTimeBacklightTimeEdit, SIGNAL(timeChanged(const QTime &)), SLOT(saveSettings()));
     connect(mUi->onBatteryDischargingCheckBox, SIGNAL(toggled(bool)), SLOT(saveSettings()));
+
+    connect(mUi->disableIdlenessFullscreenBox, SIGNAL(toggled(bool)), this, SLOT(saveSettings()));
```

Comment:
Please don't use the old connection format (above and below)! The other lines should be changed later (in another PR).